.github/workflows/refresh-ledger.yaml

```diff
@@ -87,9 +88,16 @@ jobs:
 
         echo "full-service shutdown successfully"
 
-    - name: copy ledger data.mdb to Azure Blob Storage
+    - name: copy ledger data.mdb to Azure Blob Storage - westeurope
       env:
         AZURE_STORAGE_CONNECTION_STRING: ${{ secrets.LEDGER_DB_AZURE_STORAGE_CONNECTION_STRING }}
```

Comment:
Should probably copy the existing secret into a new `LEDGER_DB_WESTEUROPE_AZURE_STORAGE_CONNECTION_STRING` secret, thus keeping the naming consistent and less ambiguous. Non-blocking.

Reply:
Fixed